src/admin/components/Actions/ListActionsCreate.tsx

```diff
@@ -0,0 +1,18 @@
+import DownloadIcon from "@mui/icons-material/GetApp";
+import { Button, CreateButton, TopToolbar } from "react-admin";
+import { When } from "react-if";
+
+interface ListActionsCreateProps {
+  onExport?: () => void;
+}
+
+const ListActionsCreate = (props: ListActionsCreateProps) => (
+  <TopToolbar>
+    <CreateButton className="filter-button-page-admin" />
+    <When condition={!!props.onExport}>
+      <Button className="button-page-admin" label="Export" startIcon={<DownloadIcon />} onClick={props.onExport} />
+    </When>
+  </TopToolbar>
+);
```

Comment:
This is fine as-is, but it might be easier to read with destructuring in the constructor:
```
const ListActionsCreate = ({ onExport }: ListActionsCreateProps) => {
 ...
}
```
(and then just use `onExport` instead of `props.onExport`

Reply:
Thanks for the suggestion! I updated the code to use destructuring in the constructor for better readability.